php-frontend/src/test/java/org/sonar/php/cfg/CfgPrinter.java

```diff
@@ -71,7 +69,9 @@ private static String blockLabel(ControlFlowGraph cfg, CfgBlock block) {
     }
 
     String stringTree = "<not supported Tree; update CfgPrinter>";
```

Comment:
can't we return `block.toString()`? (will require to update `toString` to consider label)